core/build/headers/sgx_attributes.h

```diff
@@ -40,7 +40,9 @@
 #define SGX_FLAGS_MODE64BIT      0x0000000000000004ULL     /* If set, then the enclave is 64 bit */
 #define SGX_FLAGS_PROVISION_KEY  0x0000000000000010ULL     /* If set, then the enclave has access to provision key */
 #define SGX_FLAGS_EINITTOKEN_KEY 0x0000000000000020ULL     /* If set, then the enclave has access to EINITTOKEN key */
-#define SGX_FLAGS_KSS            0x0000000000000080ULL     /* If set enclave uses KSS */
+#define SGX_FLAGS_KSS            0x0000000000000080ULL     /* If set, then the enclave uses KSS */
```

Comment:
These vendored header file updates come from, https://github.com/intel/linux-sgx/compare/sgx_2.19...sgx_2.20

Unfortunately the github web interface isn't very good for looking at the diff. I diffed the vendored headers locally with beyondcompare.